test/blackbox/run_blackbox_tests/test_cases.c

```diff
@@ -508,15 +508,15 @@ static bool test_steps_meta_conn_05(void) {
 int test_meta_conn(void) {
 	const struct CMUnitTest blackbox_group0_tests[] = {
 		cmocka_unit_test_prestate_setup_teardown(test_case_meta_conn_01, setup_test, teardown_test,
-		                (void *)&test_meta_conn_1_state),
+		(void *)&test_meta_conn_1_state),
```

Comment:
Did you run it through astyle? If so, what version of astyle do you have? The point of code formatting tools is that formatting should be consistent, but now a large part of the commit is changes in whitespace, which is exactly what code formatting should prevent...

Reply:
Hm, that's unfortunate... Can you try installing astyle 3.1 from https://packages.ubuntu.com/bionic/amd64/astyle/download? It should install on your version of Ubuntu without requiring any other packages to be updated, I think.